applications/common/server/common/__main__.py

```diff
@@ -1,23 +1,45 @@
+#!/usr/bin/env python3
+
 import os
+import logging
 
-from flask import Flask
 from flask_cors import CORS
 
 import connexion
 
 from common import encoder
 
-def main():
-    app = connexion.App(__name__, specification_dir='./openapi/')
-    app.app.config.from_object(os.environ['APP_SETTINGS'])
-    app.app.json_encoder = encoder.JSONEncoder
-    app.add_api('openapi.yaml',
-                arguments={'title': 'CH service API'},
-                pythonic_params=True)
+connexion_app = connexion.App(__name__, specification_dir='./openapi/')
+app = connexion_app.app
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.json_encoder = encoder.JSONEncoder
+
+with app.app_context():
+    # setup logging
+    gunicorn_logger = logging.getLogger("gunicorn.error")
+    app.logger.handlers = gunicorn_logger.handlers
+    app.logger.setLevel(gunicorn_logger.level)
+
     from .repository.db import open_db
+
     open_db(app)
-    cors = CORS(app.app, resources={r"/api/*": {"origins": "*"}})
-    app.run(port=8080)
+    cors = CORS(app, resources={r"/api/*": {"origins": "*"}})
+    connexion_app.add_api('openapi.yaml',
+                          arguments={'title': 'CH service API'},
+                          pythonic_params=True)
+
+    try:
+        # init_app can be defined to add behaviours to the wsgi app
+        from common import init_app
+
+        init_app(app)
+    except ImportError:
+        pass
+
+
+def main():
+    connexion_app.run(port=8080)
```

Comment:
may be better to remove the hardcoded 8080 port (set it via the FLASK_RUN_PORT or the command line parameter)